app/ptokens/migrations/0002_auto_20200607_1455.py

```diff
@@ -0,0 +1,131 @@
+        ('ptokens', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='personaltoken',
+            name='token_holders_count',
+        ),
+        migrations.RemoveField(
+            model_name='personaltoken',
+            name='token_owner_name',
+        ),
+        migrations.RemoveField(
+            model_name='personaltoken',
+            name='web3_type',
+        ),
+        migrations.RemoveField(
+            model_name='redemptiontoken',
+            name='buyer_address',
+        ),
+        migrations.RemoveField(
+            model_name='redemptiontoken',
+            name='buyer_profile',
+        ),
+        migrations.RemoveField(
+            model_name='redemptiontoken',
+            name='web3_type',
+        ),
+        migrations.AddField(
+            model_name='personaltoken',
+            name='network',
+            field=models.CharField(default='', max_length=255),
+        ),
+        migrations.AddField(
+            model_name='personaltoken',
+            name='token_symbol',
+            field=models.CharField(max_length=10, null=True),
+        ),
+        migrations.AddField(
+            model_name='personaltoken',
+            name='tx_status',
+            field=models.CharField(choices=[('na', 'na'), ('pending', 'pending'), ('success', 'success'), ('error', 'error'), ('unknown', 'unknown'), ('dropped', 'dropped')], db_index=True, default='na', max_length=9),
+        ),
+        migrations.AddField(
+            model_name='personaltoken',
+            name='value',
+            field=models.DecimalField(blank=True, decimal_places=2, default=0, max_digits=50, null=True),
+        ),
+        migrations.AddField(
+            model_name='redemptiontoken',
+            name='network',
+            field=models.CharField(default='', max_length=255),
+        ),
+        migrations.AddField(
+            model_name='redemptiontoken',
+            name='redemption_accepted',
+            field=models.DateTimeField(null=True),
+        ),
+        migrations.AddField(
+            model_name='redemptiontoken',
+            name='redemption_requester',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='redemptions', to='dashboard.Profile'),
+        ),
+        migrations.AddField(
+            model_name='redemptiontoken',
+            name='tx_status',
+            field=models.CharField(choices=[('na', 'na'), ('pending', 'pending'), ('success', 'success'), ('error', 'error'), ('unknown', 'unknown'), ('dropped', 'dropped')], db_index=True, default='na', max_length=9),
+        ),
+        migrations.AlterField(
+            model_name='personaltoken',
+            name='token_owner_profile',
+            field=models.OneToOneField(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='token_created', to='dashboard.Profile'),
```

Comment:
I'm not certain this will work OnetoOneField.

Reply:
No problem the migration `app/ptokens/migrations/0003_auto_20200607_1850.py` change it to `ForeignKey`